### ISSUE TITLE
[bug] koreader (or rather sdcv) word lookup speed seems to ignore whether dictionaries are enabled/disabled in gui

### ISSUE THREAD

**Issue by arooni** [koreader/koreader#6064]
* KOReader version: 2020.04
* Device: PW4

#### Issue
Full discussion:  [on mobileread](https://www.mobileread.com/forums/showthread.php?p=3977220#post3977220)

Essentially it seems even with dictionaries disabled in the koreader interface, they still seem to be used on lookup.  I had the Oxford English Learner's Dictionary on my kindle and whether or not it was selected in the GUI interface it slowed down lookups till it wasn't usable.  The only way to solve it was to move it out of the /data/dict directory.  Once I did, lookups became nearly instantaneous.  

tldr; lookup speed depends on whether the dictionaries are in the /data/dict, not whether or not they are selected in the koreader gui.

**Comment by poire-z** [comment]
See https://github.com/koreader/koreader/pull/3144#issuecomment-614893177
Try moving out just the `res/` subdirectory.

**Comment by arooni** [comment]
I deleted the learner dictionary when I found out it was the culprit.  :\

**Comment by poire-z** [comment]
I'd still like to know what speed you get with it without the res/ subdirectory - and be sure that's the reason.

**Comment by arooni** [comment]
Considering there are apparently 226MB of jpg in /res/ for that dictionary I'm almost positive you're right.

**Comment by poire-z** [comment]
Sorry, "most likely" and "almost positive" won't do :)
But back your dict and give us (and people like you who will open such issues in the future) facts.

**Comment by arooni** [comment]
fair point.  

https://gist.github.com/arooni/7fd8b7b4d44d20c346dd62aeb4785375

seems like 50% faster removing the res directory.

**Comment by poire-z** [comment]
OK, thanks.
And from the user point of feeling of speed: with the res/ subdirectory removed, is the title of this issue still a thing?

**Comment by arooni** [comment]
with the res directory removed lookups seem near instantaneous even with 3
dictionaries enabled.  so i'm guessing this could be closed as a non issue
now.

may today be the best day of your life,
-------
david parkinson
founder - aroonilabs.com
blog - davidparkinson.com <https://www.davidparkinson.com/>
twitter - @followdavid <https://twitter.com/followdavid>
